ttnn/ttnn/operations/binary_backward.py

```diff
@@ -186,6 +186,41 @@ def _golden_function_comparison_ops(torch_op, grad_tensor, input_tensor_a, input
     ),
 )
 
+ttnn.attach_golden_function(
+    ttnn.eq_bw,
+    golden_function=lambda grad, a, b, *args, **kwargs: _golden_function_comparison_ops(
+        torch.eq, grad, a, b, *args, **kwargs
```

Comment:
Given the these comparison ops are not differentiable and always end up with zero tensors, it does make sense to make this common code as you mentioned earlier.